app/controllers/UpdateCaseCheckYourAnswersController.scala

```diff
@@ -59,6 +62,8 @@ class UpdateCaseCheckYourAnswersController @Inject()(identify: IdentifierAction,
         updateCaseService.updateCase().flatMap {
           case Left(UpdateCaseError.InvalidCaseId) =>
             Future.successful(Redirect(controllers.routes.DisclosureNotFoundController.onLoad()))
+          case Left(UpdateCaseError.CaseAlreadyClosed) =>
+            Future.successful(Redirect(controllers.errors.routes.InformationCannotBeAddedController.onLoad(caseId)))
```

Comment:
can this be in the top level for the moment as I want to put it in the updateCase package when I create it.